contracts/cross-chain/IZNSZChainPortal.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+import { IPolygonZkEVMBridgeV2 } from "@zero-tech/zkevm-contracts/contracts/v2/interfaces/IPolygonZkEVMBridgeV2.sol";
+import { IDistributionConfig } from "../types/IDistributionConfig.sol";
+import { IZNSRootRegistrarTrunk } from "../registrar/IZNSRootRegistrarTrunk.sol";
+import { IZNSSubRegistrarTrunk } from "../registrar/IZNSSubRegistrarTrunk.sol";
+import { IZNSRegistry } from "../registry/IZNSRegistry.sol";
+import { IZNSTreasury } from "../treasury/IZNSTreasury.sol";
+import { IZNSChainResolver } from "../resolver/IZNSChainResolver.sol";
+import { IZNSAccessController } from "../access/IZNSAccessController.sol";
+
+
+interface IZNSZChainPortal is IDistributionConfig {
+
+    struct ZNSContractInput {
```

Comment:
You put this in a struct but the other contract has the same arguments, except for `IZNSAccessController`, might make more sense to use this struct as an initializer argument for both contracts and just have the one additional argument of the access controller contract be given in the `ZNSZChainPortal` contract